update-dois/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,5 @@
+eclipse.preferences.version=1
```

Comment:
This file is only valid for Eclipse and if it contains any absolute paths, it will only be useful for *you*. This file should be removed from the repo and then ignored in .gitignore

Reply:
done